README.md

```diff
@@ -408,8 +404,7 @@ as instructed in the notebook or by running the following:
 
 ```bash
 # Obtain the embeddings
-$ python scripts/extract.py esm1v_t33_650M_UR90S_1 examples/data/P62593.fasta examples/data/P62593_emb_esm1v/ \
-    --repr_layers 33 --include mean
+python scripts/extract.py esm1v_t33_650M_UR90S_1 examples/data/P62593.fasta examples/data/P62593_emb_esm1v --repr_layers 33 --include mean
```

Comment:
revert also, or improve balance line length :)

Reply:
You're right, my bad! 
I fixed them.